src/components/ResultList/Item/Item.tsx

```diff
@@ -66,20 +52,11 @@ export const Item = (props: IItemProps): ReactElement => {
 
   return (
     <Flex direction="row" as="article" border="1px" borderColor="gray.50" mb={1} borderRadius="md">
-      <Flex direction="row" className={checkBgClass} m={0}>
-        <Text color={state.matches('selected') ? 'white' : 'initial'} display={{ base: 'none', md: 'initial' }} mr={1}>
-          {index}
+      <Flex direction="row" m={0}>
```

Comment:
I believe here the style was accidentally removed. 
```suggestion
      <Flex
        direction="row"
        backgroundColor={isChecked ? 'blue.500' : 'gray.50'}
        justifyContent="center"
        alignItems="center"
        mr="2"
        px="2"
        borderLeftRadius="md"
      >
```